xml/System.IO.Pipelines/PipeOptions.xml

```diff
@@ -45,14 +45,14 @@
         <Parameter Name="useSynchronizationContext" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="pool">To be added.</param>
-        <param name="readerScheduler">To be added.</param>
-        <param name="writerScheduler">To be added.</param>
-        <param name="pauseWriterThreshold">To be added.</param>
-        <param name="resumeWriterThreshold">To be added.</param>
-        <param name="minimumSegmentSize">To be added.</param>
-        <param name="useSynchronizationContext">To be added.</param>
-        <summary>Creates a new instance of <see cref="T:System.IO.Pipelines.PipeOptions" />.</summary>
+        <param name="pool">The pool of memory blocks to be used for buffer management.</param>
+        <param name="readerScheduler">The <see cref="T:System.IO.Pipelines.PipeScheduler" /> to be used to execute <see cref="T:System.IO.Pipelines.PipeReader" /> callbacks.</param>
+        <param name="writerScheduler">The <see cref="T:System.IO.Pipelines.PipeScheduler" /> used to execute <see cref="T:System.IO.Pipelines.PipeWriter" /> callbacks.</param>
+        <param name="pauseWriterThreshold">The number of bytes in the <see cref="T:System.IO.Pipelines.Pipe" /> when <see cref="M:System.IO.Pipelines.PipeWriter.FlushAsync(System.Threading.CancellationToken)" /> starts blocking.</param>
+        <param name="resumeWriterThreshold">The number of bytes in the <see cref="T:System.IO.Pipelines.Pipe" /> when <see cref="M:System.IO.Pipelines.PipeWriter.FlushAsync(System.Threading.CancellationToken)" /> stops blocking.</param>
+        <param name="minimumSegmentSize">The minimum size of the segment requested from <paramref name="pool" />.</param>
+        <param name="useSynchronizationContext"><see langword="true" /> if asynchronous callbacks should be executed on the <see cref="T:System.Threading.SynchronizationContext" /> they were captured on; <see langword="false" /> otherwise. This takes precedence over the schedulers specified in <see cref="P:System.IO.Pipelines.PipeOptions.ReaderScheduler" /> and <see cref="P:System.IO.Pipelines.PipeOptions.WriterScheduler" />.</param>
```

Comment:
```suggestion
        <param name="useSynchronizationContext"><see langword="true" /> if asynchronous continuations should be executed on the <see cref="T:System.Threading.SynchronizationContext" /> they were captured on; <see langword="false" /> otherwise. This takes precedence over the schedulers specified in <see cref="P:System.IO.Pipelines.PipeOptions.ReaderScheduler" /> and <see cref="P:System.IO.Pipelines.PipeOptions.WriterScheduler" />.</param>
```

Reply:
@BillWagner can I have your help on the async terminology here?